### DISCUSSION TITLE
How to disable the font library

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#326] [Topic Ideas]
This would be a Snippet post that shows how to disable the font library (useful for curating the user experience when used alongside font presets):

```php
add_filter( 'block_editor_settings_all', 'devblog_block_editor_settings' );

function devblog_block_editor_settings( $settings ) {
	$settings['fontLibraryEnabled'] = false;

	return $settings;
}
```

Related:

- https://developer.wordpress.org/reference/hooks/block_editor_settings_all/
- https://developer.wordpress.org/news/2024/07/04/15-ways-to-curate-the-wordpress-editing-experience/
```

**Comment by bph** [comment]
Snippet approved, conversations continue on the issue: 
- #337

**Comment by bph** [comment]
Published https://developer.wordpress.org/news/snippets/how-to-disable-the-font-library/
